frontend/src/components/layout/Layout.module.scss

```diff
@@ -4,7 +4,7 @@
 .wrapper {
   display: flex;
   flex-direction: column;
-  height: 100%;
```

Comment:
I think we have to solve it to look as it currently does. It would be silly to leave it as is - I did not know there was a dependency here. Thanks for pointing that out. I'll take a look at the suggestion below and see if that affects any other parts of the site (if it works).